### PR TITLE
04 - Homework - Nada Alansari

### PR DESCRIPTION
**Pull request by nada0an9**

### CODE REVIEW

File: MyPlayground copy.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+var Mycity = "Medinah"
```
Comment (SalimNoor) [inline MyPlayground copy.playground/Contents.swift:3]: Try to use swift naming standards for variables. Here its recommended to have like 
**var myCity = "Madinah"**

i.e. start with small case then make first letter of each word Upper case.

**Review by SalimNoor — DISMISSED** [review]

File: MyPlayground copy.playground/Contents.swift (hunk 1 above)
Comment (azfay3) [reply MyPlayground copy.playground/Contents.swift:3]: 1. Adopt camelCase pattern. 
2. `var Mycity = "Dubi"` < is missing!

**Review by azfay3 — DISMISSED** [review]
As discussed, always adopt camelCase pattern for variables, constants & functions naming.

**Comment by azfay3** [comment]
Great job!
*Note: selectedCity exists in top of file.

**Pull request closed without merging**